Sources/PointFree/Episode/Show.swift

```diff
@@ -45,29 +48,86 @@ let episodeResponse =
     }
 )
 
-let useCreditResponse =
-  fetchEpisodeForParam
+private func episodePageData(
+  currentUser: User?,
+  collectionSlug: Episode.Collection.Slug?,
+  episode: Episode,
+  permission: EpisodePermission,
+  subscriberState: SubscriberState
+) -> Either<EpisodePageData, NewEpisodePageData> {
+  if currentUser?.isAdmin == .some(true) {
```

Comment:
admins get the new episode page, and in order to show the new episode page we have to determine if we are in the context of a collection or the context of a direct episode visit.